diff --git a/node_modules/@ecl/twig-component-file/file-translation-item.html.twig b/node_modules/@ecl/twig-component-file/file-translation-item.html.twig
index a934d0f..0671b73 100644
--- a/node_modules/@ecl/twig-component-file/file-translation-item.html.twig
+++ b/node_modules/@ecl/twig-component-file/file-translation-item.html.twig
@@ -5,20 +5,25 @@
   - "title" (string) (default: '')
   - "meta" (string) (default: '')
   - "lang" (string) (default: '')
+  - "lang_full" (string) (default: '')
   - "id" (string) (default: '')
   - "title_id" (string) (default: '')
   - "download" (object) (default to the parent download property) object of type Link
   - "download_attribute" (bool) (default to the parent download attribute)
+  - "description (string) (default:'')
 #}
 
 {# Internal properties #}
 
 {% set _lang = lang|default('') %}
+{% set _lang_full = lang_full|default('') %}
 {% set _meta = meta|default('') %}
 {% set _download = download|default({}) %}
 {% set _download_attribute = download_attribute|default(download_attribute_default) %}
 {% set _id = id|default('') %}
 {% set _title_id = title_id|default('') %}
+{% set _description = description|default('') %}
+{% set _variant = variant|default('default') %}
 
 {% set _link_id = _id ~ '-link' %}
 {% set _label_id = _link_id ~ '-label' %}
@@ -37,13 +42,36 @@
 {% endset %}
 
   <li class="ecl-file__translation-item">
-    <div class="ecl-file__translation-info">
-      {{- _title -}}
-    {% if _meta is not empty %}
-      <div class="ecl-file__translation-meta">
-        {{- _meta -}}
+    {% if _variant == 'thumbnail' %}
+      <div class="ecl-file__translation-detail">
+        {{- _title -}}
+        {% if _description is not empty %}
+          <div
+            class="ecl-file__translation-description"
+            {% if _lang is not empty %}
+              lang="{{ _lang }}"
+            {% endif %}
+          >
+            {{- _description -}}
+          </div>
+        {% endif %}
       </div>
     {% endif %}
+    <div class="ecl-file__translation-info">
+      {% if _variant != 'thumbnail' %}
+        {{- _title -}}
+      {% else %}
+        {% if _lang_full is defined and _lang_full is not empty %}
+          <div class="ecl-file__translation-language">
+            {{- _lang_full -}}
+          </div>
+        {% endif %}
+      {% endif %}
+      {% if _meta is not empty %}
+        <div class="ecl-file__translation-meta">
+          {{- _meta -}}
+        </div>
+      {% endif %}
     </div>
 
     <div class="ecl-file__translation-action">
diff --git a/node_modules/@ecl/twig-component-file/file-translations.html.twig b/node_modules/@ecl/twig-component-file/file-translations.html.twig
index 0d40c88..6dd85ba 100644
--- a/node_modules/@ecl/twig-component-file/file-translations.html.twig
+++ b/node_modules/@ecl/twig-component-file/file-translations.html.twig
@@ -2,6 +2,7 @@
 
 {#
   Parameters:
+    - "variant" (string) (default: 'default')
     - "toggle" (object) (default: {}): object of type Button
     - "id" (string) (default: '')
     - "items" (array) (default: []):
@@ -9,6 +10,7 @@
       - "meta" (string) (default: '')
       - "lang" (string) (default: '')
       - "download" (object) (default to the parent download property) object of type Link
+    - "description (string) (default:'')
 #}
 
 {# Internal properties #}
@@ -17,6 +19,8 @@
 {% set _toggle = toggle|default({}) %}
 {% set _id = id|default('') %}
 {% set _download_attribute_default = download_attribute_default|default(false) %}
+{% set _description = description|default('') %}
+{% set _variant = variant|default('default') %}
 
 {% if _items is not empty %}
   <div
@@ -39,13 +43,17 @@
     }) only %}
 
     <ul class="ecl-file__translation-list">
-    {% for key, _item in _items %}
-      {% include '@ecl/file/file-translation-item.html.twig' with _item|merge({
+      {% for key, _item in _items %}
+        {% include '@ecl/file/file-translation-item.html.twig' with _item|merge({
           id:  _id ~ '-translation-' ~ key,
           title_id: _id ~ '-title',
           download_attribute_default: _download_attribute_default,
-      }) only %}
-    {% endfor %}
+          variant: _variant
+        }) only %}
+      {% endfor %}
+      <li class="ecl-file__translation-item ecl-file__translation-description">
+        {{- _description -}}
+      </li>
     </ul>
   </div>
 {% endif %}
diff --git a/node_modules/@ecl/twig-component-file/file.html.twig b/node_modules/@ecl/twig-component-file/file.html.twig
index d18c512..6368be9 100644
--- a/node_modules/@ecl/twig-component-file/file.html.twig
+++ b/node_modules/@ecl/twig-component-file/file.html.twig
@@ -32,6 +32,7 @@
       - "meta" (string) (default: '')
       - "lang" (string) (default: '')
       - "download" (object) (default to the parent download property) object of type Link
+    - "description (string) (default:'')
   - "extra_classes" (optional) (string) (default: '')
   - "extra_attributes" (optional) (array) (default: []) [{
     - "name" (string) Attribute name, eg. 'data-test'
@@ -205,6 +206,7 @@
   {% include '@ecl/file/file-translations.html.twig' with _translation|merge({
     id: _id,
     download_attribute_default: _download_attribute,
+    variant: _variant|default('default'),
   }) only %}
 {% endif %}
 </div>
