diff --git a/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig b/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
index 761c4da..7f8f967 100644
--- a/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
+++ b/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
@@ -6,6 +6,8 @@
     - "section_class_name" (string) (default: '')
     - "logo" (EU only)
       - "alt" (string) alt attribute for the logo link
+      - "aria_label" (string) aria-label attribute for the logo link
+      - "title" (string) title attribute for the logo img
       - "path" (string) logo link path
       - "language" (string) Language code
       - "src_mobile" (string) Path the mobile logo
@@ -101,7 +103,7 @@
               path: _sections.logo.path,
               label: _label,
               type: 'standalone',
-              aria_label: _sections.logo.title
+              aria_label: _sections.logo.aria_label
             },
             extra_classes: 'ecl-site-footer__logo-link'
           } only %}
