diff --git a/node_modules/@ecl/mega-menu/mega-menu-item.html.twig b/node_modules/@ecl/mega-menu/mega-menu-item.html.twig
index 08edc10..0ecc1f7 100644
--- a/node_modules/@ecl/mega-menu/mega-menu-item.html.twig
+++ b/node_modules/@ecl/mega-menu/mega-menu-item.html.twig
@@ -191,10 +191,10 @@
             size: 'xs',
           },
           extra_classes: _info.link.extra_classes|default('') ~ ' ecl-mega-menu__info-link',
-          extra_attributes: [{
+          extra_attributes: _info.link.extra_attributes|default([])|merge([{
             name: 'aria-describedby',
             value: _info_title_id,
-          }],
+          }]),
         }) only %}
       {%- endif -%}
       </div>
