diff --git a/node_modules/@ecl/twig-component-video/video.html.twig b/node_modules/@ecl/twig-component-video/video.html.twig
index aed3726..0318a5f 100644
--- a/node_modules/@ecl/twig-component-video/video.html.twig
+++ b/node_modules/@ecl/twig-component-video/video.html.twig
@@ -38,6 +38,7 @@
 {% set _title = title|default('') %}
 {% set _sr_video_player = sr_video_player|default('') %}
 {% set _sr_video_label = sr_video_label|default('') %}
+{% set _extra_attributes = '' %}
 
 {% if _controls %}
   {% set _extra_attributes = 'controls' %}
