patch-package
--- a/node_modules/@ecl-twig/ec-component-table/ecl-table.html.twig
+++ b/node_modules/@ecl-twig/ec-component-table/ecl-table.html.twig
@@ -108,7 +108,7 @@
         {% set _row_extra_attributes = row.extra_attributes %}
       {% endif %}
       <tr class="{{ _row_css_class }}"{{ _row_extra_attributes }}>
-        {% for cell in row %}
+        {% for cell in row.cells %}
           {% set _cell_css_class = 'ecl-table__cell' %}
           {% set _cell_attribute = cell['data-ecl-table-header'] is not empty ? 'data-ecl-table-header="' ~ cell['data-ecl-table-header']|e('html_attr') ~ '"' : '' %}
           {% if cell.group %}
