patch-package
--- a/node_modules/@ecl-twig/ec-component-card/ecl-card.html.twig
+++ b/node_modules/@ecl-twig/ec-component-card/ecl-card.html.twig
@@ -91,7 +91,7 @@
       <div class="ecl-card__image" {{ _card_img_aria_label|raw }} role="img" style="background-image:url('{{ _card.image.src }}')"></div>
     {% endif %}
     {% if _card.meta is not empty %}
-      <div class="ecl-card__meta">{{ _card.meta|join(" | ") }}</div>
+      <div class="ecl-card__meta">{{ _card.meta|safe_join(" | ") }}</div>
     {% endif %}
     <h1 class="ecl-card__title">
       {%- if _card.title.path %}
