patch-package
--- a/node_modules/@ecl-twig/ec-component-search-form/ecl-search-form.html.twig
+++ b/node_modules/@ecl-twig/ec-component-search-form/ecl-search-form.html.twig
@@ -45,31 +45,33 @@
 
 {# Print the result #}
 
-<form class="{{ _css_class }}" role="search"{{ _extra_attributes|raw }}>
-  {% if _text_input is not empty %}
-    {% include '@ecl-twig/ec-component-text-input/ecl-text-input.html.twig' with _text_input|merge({
-      type: 'search',
-      extra_label_classes: 'ecl-search-form__label',
-      extra_classes: 'ecl-search-form__text-input'
-    }) only %}
-  {% endif %}
-  {% if _button is not empty %}
-    {% include '@ecl-twig/ec-component-button/ecl-button.html.twig' with _button|merge({
-      variant: 'search',
-      icon: {
-        path: _button.icon.path,
-        type: 'general',
-        name: 'search',
-        size: 'xs',
-      },
-      extra_classes: 'ecl-search-form__button',
-      extra_attributes: [{name: 'aria-label', value: 'Search'}],
-      _compliance_: _compliance,
-      _compliance_inner_check_: true
-    }) only %}
-  {% endif %}
-  {%- block extra_form_elements _extra_form_elements|raw -%}
-</form>
+{% block search_form %}
+  <form class="{{ _css_class }}" role="search"{{ _extra_attributes|raw }}>
+    {% if _text_input is not empty %}
+      {% include '@ecl-twig/ec-component-text-input/ecl-text-input.html.twig' with _text_input|merge({
+        type: 'search',
+        extra_label_classes: 'ecl-search-form__label',
+        extra_classes: 'ecl-search-form__text-input'
+      }) only %}
+    {% endif %}
+    {% if _button is not empty %}
+      {% include '@ecl-twig/ec-component-button/ecl-button.html.twig' with _button|merge({
+        variant: 'search',
+        icon: {
+          path: _button.icon.path,
+          type: 'general',
+          name: 'search',
+          size: 'xs',
+        },
+        extra_classes: 'ecl-search-form__button',
+        extra_attributes: [{name: 'aria-label', value: 'Search'}],
+        _compliance_: _compliance,
+        _compliance_inner_check_: true
+      }) only %}
+    {% endif %}
+    {%- block extra_form_elements _extra_form_elements|raw -%}
+  </form>
+{% endblock search_form %}
 
 {# Validation #}
 
