patch-package
--- a/node_modules/@ecl-twig/ec-component-page-header-standardised/ecl-page-header-standardised.html.twig
+++ b/node_modules/@ecl-twig/ec-component-page-header-standardised/ecl-page-header-standardised.html.twig
@@ -49,6 +49,11 @@
       _compliance_inner_check_: true
     }) only %}
   {% endif %}
+  {% if composition is not empty %}
+    <div class="ecl-page-header-standardised__composition">
+      {{- composition|raw -}}
+    </div>
+  {% endif %}
   {% if _meta is not empty %}
     <div class="ecl-page-header-standardised__meta">
       {{- _meta -}}
