patch-package
--- a/node_modules/@ecl-twig/ec-component-page-header/ecl-page-header.html.twig
+++ b/node_modules/@ecl-twig/ec-component-page-header/ecl-page-header.html.twig
@@ -76,16 +76,20 @@
   <div class="ecl-container">
   {% if _breadcrumb is not empty %}
     {% include '@ecl-twig/ec-component-breadcrumb/ecl-breadcrumb.html.twig' with _breadcrumb|merge({
-      extra_classes: 'ecl-page-header__breadcrumb',
+      extra_classes: not composition ? 'ecl-page-header__breadcrumb',
       _compliance_: _compliance,
       _compliance_inner_check_: true
     }) only %}
   {% endif %}
   {% if composition is defined and composition is not empty %}
-    {{- composition|raw -}}
+    <div class="ecl-u-mb-xl ecl-u-border-bottom ecl-u-border-color-white">
+      {{- composition|raw -}}
+    </div>
   {% endif %}
   {% if _meta is not empty %}
-    <div class="ecl-page-header__meta-list">{{ _meta }}</div>
+    <div class="ecl-page-header__title-wrapper">
+      <div class="ecl-page-header__meta-list">{{ _meta }}</div>
+    </div>
   {% endif %}
   {% if _title_wrapper %}
     <div class="ecl-page-header__title-wrapper">
@@ -95,7 +99,7 @@
       <p class="ecl-page-header__slogan">{{ _slogan|raw }}</p>
     {% endif %}
     {% if _description is not empty %}
-      <p class="ecl-page-header__description">{{ _description|raw }}</p>
+      <p class="ecl-page-header__description">{{ _description }}</p>
     {% endif %}
   {% if _title_wrapper %}
     </div>
