patch-package
--- a/node_modules/@ecl-twig/ec-component-menu/ecl-menu.html.twig
+++ b/node_modules/@ecl-twig/ec-component-menu/ecl-menu.html.twig
@@ -152,8 +152,8 @@
           </span>
         </button>
       </header>
-      <ul class="ecl-menu__list">
       {% if items is defined and items is not empty and items is iterable %}
+      <ul class="ecl-menu__list">
         {% for item in items %}
         {% if item is defined and item is not empty %}
           {% include '@ecl-twig/ec-component-menu/ecl-menu-item.html.twig' with {
@@ -163,8 +163,8 @@
           } only %}
         {% endif %}
         {% endfor %}
-      {% endif %}
       </ul>
+      {% endif %}
     </section>
   </div>
 </nav>
