patch-package
--- a/node_modules/@ecl-twig/ec-component-timeline/ecl-timeline.html.twig
+++ b/node_modules/@ecl-twig/ec-component-timeline/ecl-timeline.html.twig
@@ -59,7 +59,7 @@ Parameters:
 {% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
   {% for attr in extra_attributes %}
     {% if attr.value is defined %}
-      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e ~ '"' %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
     {% else %}
       {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
     {% endif %}
@@ -81,7 +81,7 @@ Parameters:
         {% if _item.title is defined and _item.title is not empty %}
         <div class="ecl-timeline2__title">{{ _item.title }}</div>
         {% endif %}
-        <div class="ecl-timeline2__content">{{ _item.content|default('')|raw }}</div>
+        <div class="ecl-timeline2__content">{{ _item.content|default('') }}</div>
       </li>
       {# Display the show more button if required. #}
       {% if (_hide_amount > 0 and loop.revindex == _bottom_amount + 1) or (_hide_amount > 0 and _hide_to == 0 and loop.revindex == 1) %}
