patch-package
--- a/node_modules/@ecl-twig/ec-component-fact-figures/ecl-fact-figures.html.twig
+++ b/node_modules/@ecl-twig/ec-component-fact-figures/ecl-fact-figures.html.twig
@@ -79,7 +79,7 @@
     {% endif %}
     {% if item.description is defined and item.description is not empty %}
       <div class="ecl-fact-figures__description">
-        {{- item.description|raw -}}
+        {{- item.description -}}
       </div>
     {% endif %}
     </div>
