patch-package
--- a/node_modules/@ecl-twig/ec-component-page-header-core/ecl-page-header-core.html.twig
+++ b/node_modules/@ecl-twig/ec-component-page-header-core/ecl-page-header-core.html.twig
@@ -62,6 +62,11 @@
       _compliance_inner_check_: true
     }) only %}
   {% endif %}
+  {% if composition is not empty %}
+    <div class="ecl-page-header-core__composition">
+      {{- composition|raw -}}
+    </div>
+  {% endif %}
   {% if _meta is not empty %}
     <div class="ecl-page-header-core__meta">
       {{- _meta -}}
