patch-package
--- a/node_modules/@ecl-twig/ec-component-media-container/ecl-media-container.html.twig
+++ b/node_modules/@ecl-twig/ec-component-media-container/ecl-media-container.html.twig
@@ -96,9 +96,11 @@
     src="{{ _image }}"
   />
 {% endif %}
-  <figcaption class="ecl-media-container__caption">
-    {{- _description -}}
-  </figcaption>
+  {% if _description is not empty %}
+    <figcaption class="ecl-media-container__caption">
+      {{- _description -}}
+    </figcaption>
+  {% endif %}
 </figure>
 
 {% endspaceless %}
