patch-package
--- a/node_modules/@ecl-twig/ec-component-card/ecl-card.html.twig
+++ b/node_modules/@ecl-twig/ec-component-card/ecl-card.html.twig
@@ -102,7 +102,7 @@
       <div class="ecl-card__image" {{ _card_img_aria_label|raw }} role="img" style="background-image:url('{{ _card.image.src }}')"></div>
     {% endif %}
     {% if _card.meta is not empty %}
-      <div class="ecl-card__meta">{{ _card.meta|join(" | ") }}</div>
+      <div class="ecl-card__meta">{{ _card.meta|safe_join(" | ") }}</div>
     {% endif %}
     <h1 class="ecl-card__title">
       {%- if _card.title.path %}
@@ -142,18 +142,19 @@
   {% if _card.description is not empty or _card.links is not empty %}
   </div>
   {% endif %}
-  <footer class="ecl-card__footer">
-    {% if _card.infos is defined and _card.infos is not empty and _card.infos is iterable %}
-      <ul class="ecl-card__info-container">
-        {% for info in _card.infos %}
-          {% if info.icon is defined and info.icon is not empty %}
-            {% set _icon = info.icon|merge({ size: 'xs' }) %}
-            {% if _icon_path is not empty %}
-              {% set _icon = _icon|merge({ path: _icon_path }) %}
-            {% else %}
-              {% set _icon_path = _icon.path|default('') %}
+  {% if _card.infos is defined and _card.infos is not empty or _card.tags is defined and _card.tags is not empty %}
+    <footer class="ecl-card__footer">
+      {% if _card.infos is defined and _card.infos is not empty and _card.infos is iterable %}
+        <ul class="ecl-card__info-container">
+          {% for info in _card.infos %}
+            {% if info.icon is defined and info.icon is not empty %}
+              {% set _icon = info.icon|merge({ size: 'xs' }) %}
+              {% if _icon_path is not empty %}
+                {% set _icon = _icon|merge({ path: _icon_path }) %}
+              {% else %}
+                {% set _icon_path = _icon.path|default('') %}
+              {% endif %}
             {% endif %}
-          {% endif %}
           <li class="ecl-card__info-item">
             {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
               icon: _icon|default({}),
@@ -163,29 +164,30 @@
             <span class="ecl-card__info-label">{{ info.label }}</span>
           </li>
         {% endfor %}
-      </ul>
-    {% endif %}
-    {% if _card.tags is defined and _card.tags is not empty and _card.tags is iterable %}
-      <ul class="ecl-card__tag-container">
-        {% for tag in _card.tags %}
-          <li class="ecl-card__tag-item">
-            {% include '@ecl-twig/ec-component-tag/ecl-tag.html.twig' with {
-              tag: tag,
-              extra_classes: 'ecl-card__tag',
-              _compliance_: _compliance,
-              _compliance_inner_check_: true,
-              icon_path: _icon_path
-            } only %}
-          </li>
+        </ul>
+      {% endif %}
+      {% if _card.tags is defined and _card.tags is not empty and _card.tags is iterable %}
+        <ul class="ecl-card__tag-container">
+          {% for tag in _card.tags %}
+            <li class="ecl-card__tag-item">
+              {% include '@ecl-twig/ec-component-tag/ecl-tag.html.twig' with {
+                tag: tag,
+                extra_classes: 'ecl-card__tag',
+                _compliance_: _compliance,
+                _compliance_inner_check_: true,
+                icon_path: _icon_path
+              } only %}
+            </li>
+          {% endfor %}
+        </ul>
+      {% endif %}
+      {% if _card.lists is defined and _card.lists is not empty and _card.lists is iterable %}
+        {% for _list in _card.lists %}
+          {% include '@ecl-twig/ec-component-description-list/ecl-description-list.html.twig' with _list|merge({ extra_classes: 'ecl-card__list'}) only %}
         {% endfor %}
-      </ul>
-    {% endif %}
-    {% if _card.lists is defined and _card.lists is not empty and _card.lists is iterable %}
-      {% for _list in _card.lists %}
-        {% include '@ecl-twig/ec-component-description-list/ecl-description-list.html.twig' with _list|merge({ extra_classes: 'ecl-card__list'}) only %}
-      {% endfor %}
-    {% endif %}
-  </footer>
+      {% endif %}
+    </footer>
+  {% endif %}
 </article>
 
 {# Validation #}
