patch-package
--- a/node_modules/@ec-europa/ecl-pagers/ecl-pagers.twig
+++ b/node_modules/@ec-europa/ecl-pagers/ecl-pagers.twig
@@ -1,26 +1,37 @@
 {#
+  - "previous" (array): Page link for the previous page; not present on the first page
+  - "first" (array): Page link for the first page; not present on the first page of results.
+  - "last" (array): Page link for the last page; not present on the last page of results.
+  - "next" (array): Page link for the next page; not present on the last page of results.
+    Each of the page link variables which are listed above contain following elements:
+    - href: URL with appropriate query parameters.
+    - text: The visible text used for the page link.
+    - title: The text used for filling the 'title' property of a given tag.
+  - "pages" (array): List of the pager page links.
+    The list is keyed with ordered page numbers and each item contatin following elements:
+    - href: URL with appropriate query parameters.
+    - text: The visible text used for the page link.
+    - title: The text used for filling the 'title' property of a given tag.
   - "current_page" (int): page currently displayed (default: 1)
   - "max_page" (int): total number of pages (default: 0)
-  - "fragment_url" (string): url parameter to identify current page (default: '/page')
-  - "title_previous" (string): helper when hovering previous page button (default: 'Go to previous page')
-  - "title_page" (string): helper when hovering a page button (default: 'Go to page')
-  - "title_next" (string): helper when hovering next page button (default: 'Go to next page')
-  - "label_previous" (string): previous button label (default: '‹ Previous')
   - "label_page" (string): page label displayed on small screens (default: 'Page')
-  - "label_next" (string): next button label (default: 'Next ›')
   - "extra_classes" (string): extra CSS classes to be added
   - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
 #}
 
 {% set current_page = current_page|default(1) %}
 {% set max_page = max_page|default(0) %}
-{% set fragment_url = fragment_url|default('/page') %}
-{% set title_previous = title_previous|default('Go to previous page') %}
-{% set title_page = title_page|default('Go to page') %}
-{% set title_next = title_next|default('Go to next page') %}
-{% set label_previous = label_previous|default('‹ Previous') %}
 {% set label_page = label_page|default('Page') %}
-{% set label_next = label_next|default('Next ›') %}
+{% set previous = previous|default({})|merge({
+  'title': previous.title|default('Go to previous page'),
+  'text': previous.text|default('‹ Previous')
+})
+%}
+{% set next = next|default({})|merge({
+  'title': next.title|default('Go to next page'),
+  'text': next.text|default('Next ›')
+})
+%}
 
 {# Internal properties #}
 
@@ -43,76 +54,63 @@
 
 <nav class="ecl-pager__wrapper" aria-label="pagination">
   <ul class="{{ _css_class }}" {{ _extra_attributes|raw }}>
+    {# Print the previous page link. #}
     {% if current_page > 1 %}
-    <li class="ecl-pager__item ecl-pager__item--previous">
-      <a class="ecl-pager__link" title="{{ title_previous }}" href="{{ fragment_url }}={{ current_page - 1 }}">{{ label_previous }}</a>
-    </li>
+      <li class="ecl-pager__item ecl-pager__item--previous">
+        <a class="ecl-pager__link" title="{{ previous.title }}" href="{{ previous.href }}">{{ previous.text }}</a>
+      </li>
     {% endif %}
 
-    {% if current_page > 1 %}
-    <li class="ecl-pager__item ecl-pager__item--first">
-      <a class="ecl-pager__link" title="{{ title_page }} 1" href="{{ fragment_url }}=1">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>1
-      </a>
-    </li>
+    {# Print the first page link. #}
+    {% if first %}
+      <li class="ecl-pager__item ecl-pager__item--first">
+        <a class="ecl-pager__link" title="{{ first.title }}" href="{{ first.href }}">
+          <span class="ecl-u-sr-only" >{{ first.text }}</span>
+          {{ first.text }}
+        </a>
+      </li>
     {% endif %}
 
+    {# Print the left ellipsis. #}
     {% if current_page > 4 %}
-    <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
-    {% endif %}
-
-    {% if current_page > 3 %}
-    <li class="ecl-pager__item">
-      <a class="ecl-pager__link" title="{{ title_page }} {{ current_page - 2 }}" href="{{ fragment_url }}={{ current_page - 2 }}">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ current_page - 2 }}
-      </a>
-    </li>
-    {% endif %}
-
-    {% if current_page > 2 %}
-    <li class="ecl-pager__item">
-      <a class="ecl-pager__link" title="{{ title_page }} {{ current_page - 1 }}" href="{{ fragment_url }}={{ current_page - 1 }}">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ current_page - 1 }}
-      </a>
-    </li>
-    {% endif %}
-
-    <li class="ecl-pager__item ecl-pager__item--current" aria-current="page">
-      <span class="ecl-pager__item-text" >{{ label_page }}</span> {{ current_page }}
-    </li>
-
-    {% if max_page - current_page > 1 %}
-    <li class="ecl-pager__item">
-      <a class="ecl-pager__link" title="{{ title_page }} {{ current_page + 1 }}" href="{{ fragment_url }}={{ current_page + 1 }}">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ current_page + 1 }}
-      </a>
-    </li>
-    {% endif %}
-
-    {% if max_page - current_page > 2 %}
-    <li class="ecl-pager__item">
-      <a class="ecl-pager__link" title="{{ title_page }} {{ current_page + 2 }}" href="{{ fragment_url }}={{ current_page + 2 }}">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ current_page + 2 }}
-      </a>
-    </li>
+      <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
     {% endif %}
 
+    {# Print page links. #}
+    {% for key, page in pages %}
+      {# Print the current page element. #}
+      {% if current_page == key %}
+        <li class="ecl-pager__item ecl-pager__item--current" aria-current="page">
+          <span class="ecl-pager__item-text" >{{ label_page }} </span>{{ page.text }}
+        </li>
+      {% else %}
+        <li class="ecl-pager__item">
+          <a class="ecl-pager__link" title="{{ page.title }}" href="{{ page.href }}">
+            <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ page.text }}
+          </a>
+        </li>
+      {% endif %}
+    {% endfor %}
+
+    {# Print the right 'ellipsis'. #}
     {% if max_page - current_page > 3 %}
-    <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
+      <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
     {% endif %}
 
-    {% if current_page != max_page %}
-    <li class="ecl-pager__item ecl-pager__item--last">
-      <a class="ecl-pager__link" title="{{ title_page }} {{ max_page }}" href="{{ fragment_url }}={{ max_page }}">
-        <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ max_page }}
-      </a>
-    </li>
+    {# Print the last page link. #}
+    {% if last %}
+      <li class="ecl-pager__item ecl-pager__item--last">
+        <a class="ecl-pager__link" title="{{ last.title }}" href="{{ last.href }}">
+          <span class="ecl-u-sr-only" >{{ last.text }} </span>{{ last.text }}
+        </a>
+      </li>
     {% endif %}
 
+    {# Print the next page link. #}
     {% if current_page != max_page %}
-    <li class="ecl-pager__item ecl-pager__item--next">
-      <a class="ecl-pager__link" title="{{ title_next }}" href="{{ fragment_url }}={{ current_page + 1 }}">{{ label_next }}</a>
-    </li>
+      <li class="ecl-pager__item ecl-pager__item--next">
+        <a class="ecl-pager__link" title="{{ next.title }}" href="{{ next.href }}">{{ next.text }}</a>
+      </li>
     {% endif %}
   </ul>
 </nav>
