patch-package
--- a/node_modules/@ec-europa/ecl-buttons/ecl-buttons.twig
+++ b/node_modules/@ec-europa/ecl-buttons/ecl-buttons.twig
@@ -47,7 +47,7 @@
 {# Print result #}
 
 {% if _type == 'link' %}
-  <a href="{{ _href }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block link_label_block _label %}</a>
+  <a href="{{ _href }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block link_label_block _label|raw %}</a>
 {% else %}
-  <button class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block button_label_block _label %}</button>
+  <button class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block button_label_block _label|raw %}</button>
 {% endif %}
