diff --git a/Crawler.php b/Crawler.php
index 21f7fab..61b00e6 100644
--- a/Crawler.php
+++ b/Crawler.php
@@ -560,19 +560,33 @@ class Crawler implements \Countable, \IteratorAggregate
     }
 
     /**
-     * Returns the node value of the first node of the list.
+     * Returns the text of the first node of the list.
+     *
+     * Pass true as the 2nd argument to normalize whitespaces.
+     *
+     * @param mixed $default When provided and the current node is empty, this value is returned and no exception is thrown
      *
      * @return string The node value
      *
      * @throws \InvalidArgumentException When current node is empty
      */
-    public function text()
+    public function text(/* $default = null, $normalizeWhitespace = true */)
     {
         if (!$this->nodes) {
+            if (0 < \func_num_args()) {
+                return func_get_arg(0);
+            }
+
             throw new \InvalidArgumentException('The current node list is empty.');
         }
 
-        return $this->getNode(0)->nodeValue;
+        $text = $this->getNode(0)->nodeValue;
+
+        if (\func_num_args() > 1 && func_get_arg(1)) {
+            return trim(preg_replace('/(?:\s{2,}+|[^\S ])/', ' ', $text));
+        }
+
+        return $text;
     }
 
     /**
