patch-package
--- a/node_modules/@ecl/ec-component-dialog/ec-component-dialog.twig
+++ b/node_modules/@ecl/ec-component-dialog/ec-component-dialog.twig
@@ -32,7 +32,7 @@
 {# Internal properties #}
 
 {% set _dialog_id = dialog_id|default('ecl-dialog') %}
-{% set _dialog_aria_hidden = dialog_aria_hidden|default(true) %}
+{% set _dialog_aria_hidden = dialog_aria_hidden|default('true') %}
 
 {% set _dialog_title = {
   value: 'Dialog',
