patch-package
--- a/node_modules/@ecl/ec-component-list-item/ec-component-list-item.twig
+++ b/node_modules/@ecl/ec-component-list-item/ec-component-list-item.twig
@@ -70,24 +70,33 @@
       </div>
     {% endif %}
     <div class="ecl-list-item__body">
-      {% if _variant is empty or _variant == 'thumbnail' %}
+      {% if (_variant is empty or _variant == 'thumbnail') and _metas is not empty %}
         <div class="ecl-list-item__meta">
           {% block meta_block %}
             {% include '@ecl/ec-component-meta' with { metas: _metas } only %}
           {% endblock %}
         </div>
       {% endif %}
-      <h3 class="ecl-list-item__title ecl-heading ecl-heading--h3">{{ title }}</h3>
-      <p class="ecl-list-item__detail ecl-paragraph">
-        {% block detail_block %}
-          {{ _detail }}
-        {% endblock %}
-      </p>
-      <div class="ecl-list-item__action">
-        {% block action_block %}
-          {{ _action }}
-        {% endblock %}
-      </div>
+
+      {% if (title is not empty) %}
+        <h3 class="ecl-list-item__title ecl-heading ecl-heading--h3">{{ title }}</h3>
+      {% endif %}
+
+      {% if (_detail is not empty) %}
+        <div class="ecl-list-item__detail ecl-paragraph">
+          {% block detail_block %}
+            {{ _detail }}
+          {% endblock %}
+        </div>
+      {% endif %}
+
+      {% if (_action is not empty) %}
+        <div class="ecl-list-item__action">
+          {% block action_block %}
+            {{ _action }}
+          {% endblock %}
+        </div>
+      {% endif %}
     </div>
     {% if _secondary_image is not empty %}
     <div class="ecl-list-item__secondary">
