patch-package
--- a/node_modules/@ecl/generic-component-form-radio/generic-component-form-radio.twig
+++ a/node_modules/@ecl/generic-component-form-radio/generic-component-form-radio.twig
@@ -53,7 +53,7 @@
 
 {# Print the result #}
 
-{% embed '@ecl/' ~ _system ~ '-component-form-label' with { 'for': _id, 'extra_classes': _css_class, 'extra_attributes': _extra_attributes, _id: _id, _value: _value, _name: _name, _label: _label, _checked: _checked, _is_disabled: _is_disabled, _system: _system } only %}
+{% embed '@ecl/' ~ _system ~ '-component-form-label' with { 'for': _id, 'extra_classes': _css_class, 'extra_attributes': _extra_attributes, _id: _id, _value: _value, _name: _name, _label: _label, _checked: _checked, _is_disabled: _is_disabled, _system: _system } %}
   {% block label %}{% spaceless %}
     <input class="ecl-radio__input ecl-u-sr-only" type="radio" id="{{ _id }}" value="{{ _value }}" name="{{ _name }}"{{ _checked ? ' checked' : '' }}{{ (_is_disabled is defined and _is_disabled == true) ? ' disabled' : '' }} />
     <span class="ecl-radio__label">{{ _label }}</span>
