patch-package
--- a/node_modules/@ecl/generic-component-button/generic-component-button.twig
+++ a/node_modules/@ecl/generic-component-button/generic-component-button.twig
@@ -54,7 +54,7 @@
 {# Print the result #}
 
 {% if _type == 'link' %}
-  <a href="{{ _href }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block link_label_block _label %}</a>
+  <a href="{{ _href }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block link_label_block _label|raw %}</a>
 {% else %}
-  <button class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block button_label_block _label %}</button>
+  <button class="{{ _css_class }}"{{ _extra_attributes|raw }}>{% block button_label_block _label|raw %}</button>
 {% endif %}
