patch-package
--- a/node_modules/@ecl/generic-component-pager/generic-component-pager.twig
+++ b/node_modules/@ecl/generic-component-pager/generic-component-pager.twig
@@ -1,15 +1,22 @@
 {#
   Parameters:
     - "aria_label" (string) (default: 'Pagination navigation')
+    - "previous" (array): Page link for the previous page; not present on the first page
+    - "first" (array): Page link for the first page; not present on the first page of results.
+    - "last" (array): Page link for the last page; not present on the last page of results.
+    - "next" (array): Page link for the next page; not present on the last page of results.
+      Each of the page link variables which are listed above contain following elements:
+        - href: URL with appropriate query parameters.
+        - text: The visible text used for the page link.
+        - title: The text used for filling the 'title' property of a given tag.
+    - "pages" (array): List of the pager page links.
+      The list is keyed with ordered page numbers and each item contatin following elements:
+        - href: URL with appropriate query parameters.
+        - text: The visible text used for the page link.
+        - title: The text used for filling the 'title' property of a given tag.
     - "current_page" (integer)  (default: 1): page currently displayed
     - "max_page" (integer) (default: 0): total number of pages
-    - "fragment_url" (string) (default: '/page'): url parameter to identify current page
-    - "title_previous" (string) (default: 'Go to previous page'): helper when hovering previous page button
-    - "title_page" (string) (default: 'Go to page'): helper when hovering a page button
-    - "title_next" (string) (default: 'Go to next page'): helper when hovering next page button
-    - "label_previous" (string) (default: '‹ Previous')
     - "label_page" (string) (default: 'Page'): page label displayed on small screens
-    - "label_next" (string)  (default: 'Next ›')
     - "extra_classes" (string) (default: '')
     - "extra_attributes" (array) (default: []): format: [
         {
@@ -22,17 +29,21 @@

 {# Internal properties #}

-{% set _aria_label = aria_label|default('Pagination navigation') %}
-{% set _current_page = current_page|default(1) %}
-{% set _max_page = max_page|default(0) %}
-{% set _fragment_url = fragment_url|default('/page') %}
-{% set _title_previous = title_previous|default('Go to previous page') %}
-{% set _title_page = title_page|default('Go to page') %}
-{% set _title_next = title_next|default('Go to next page') %}
-{% set _label_previous = label_previous|default('‹ Previous') %}
-{% set _label_page = label_page|default('Page') %}
-{% set _label_next = label_next|default('Next ›') %}
-{% set _css_class = 'ecl-pager' %}
+{% set aria_label = aria_label|default('Pagination navigation') %}
+{% set current_page = current_page|default(1) %}
+{% set max_page = max_page|default(0) %}
+{% set label_page = label_page|default('Page') %}
+{% set previous = previous|default({})|merge({
+    'title': previous.title|default('Go to previous page'),
+    'text': previous.text|default('‹ Previous')
+  })
+%}
+{% set next = next|default({})|merge({
+    'title': next.title|default('Go to next page'),
+    'text': next.text|default('Next ›')
+  })
+%}
+{% set _css_class = 'ecl-pager__list' %}
 {% set _extra_attributes = '' %}

 {# Internal logic - Process properties #}
@@ -49,67 +60,65 @@

 {# Print the result #}

-<nav class="{{ _css_class }}" aria-label="{{ _aria_label }}"{{ _extra_attributes|raw }}>
-  <ul class="ecl-pager__list">
-    {% if _current_page > 1 %}
+<nav class="ecl-pager" aria-label="{{ aria_label }}">
+  <ul class="{{ _css_class }}" {{ _extra_attributes|raw }}>
+    {# Print the previous page link. #}
+    {% if current_page > 1 %}
       <li class="ecl-pager__item ecl-pager__item--previous">
-        <a class="ecl-pager__link" title="{{ _title_previous }}" href="{{ _fragment_url }}={{ _current_page - 1 }}">{{ _label_previous }}</a>
+        <a class="ecl-pager__link" title="{{ previous.title }}" href="{{ previous.href }}">{{ previous.text }}</a>
       </li>
     {% endif %}
-    {% if _current_page > 1 %}
+
+    {# Print the first page link. #}
+    {% if first %}
       <li class="ecl-pager__item ecl-pager__item--first">
-        <a class="ecl-pager__link" title="{{ _title_page }} 1" href="{{ _fragment_url }}=1">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>1
+        <a class="ecl-pager__link" title="{{ first.title }}" href="{{ first.href }}">
+          <span class="ecl-u-sr-only" >{{ first.text }}</span>
+          {{ first.text }}
         </a>
       </li>
     {% endif %}
-    {% if _current_page > 4 %}
+
+    {# Print the left ellipsis. #}
+    {% if current_page > 4 %}
       <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
     {% endif %}
-    {% if _current_page > 3 %}
-      <li class="ecl-pager__item">
-        <a class="ecl-pager__link" title="{{ _title_page }} {{ _current_page - 2 }}" href="{{ _fragment_url }}={{ _current_page - 2 }}">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>{{ _current_page - 2 }}
-        </a>
-      </li>
-    {% endif %}
-    {% if _current_page > 2 %}
-      <li class="ecl-pager__item">
-        <a class="ecl-pager__link" title="{{ _title_page }} {{ _current_page - 1 }}" href="{{ _fragment_url }}={{ _current_page - 1 }}">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>{{ _current_page - 1 }}
-        </a>
-      </li>
-    {% endif %}
-    <li class="ecl-pager__item ecl-pager__item--current" aria-current="true">
-      <span class="ecl-pager__item-text" >{{ _label_page }}</span> {{ _current_page }}
-    </li>
-    {% if _max_page - _current_page > 1 %}
-      <li class="ecl-pager__item">
-        <a class="ecl-pager__link" title="{{ _title_page }} {{ _current_page + 1 }}" href="{{ _fragment_url }}={{ _current_page + 1 }}">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>{{ _current_page + 1 }}
-        </a>
-      </li>
-    {% endif %}
-    {% if _max_page - _current_page > 2 %}
-      <li class="ecl-pager__item">
-        <a class="ecl-pager__link" title="{{ _title_page }} {{ _current_page + 2 }}" href="{{ _fragment_url }}={{ _current_page + 2 }}">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>{{ _current_page + 2 }}
-        </a>
-      </li>
-    {% endif %}
-    {% if _max_page - _current_page > 3 %}
+
+    {# Print page links. #}
+    {% for key, page in pages %}
+      {# Print the current page element. #}
+      {% if current_page == key %}
+        <li class="ecl-pager__item ecl-pager__item--current" aria-current="page">
+          <span class="ecl-pager__item-text" >{{ label_page }} </span>{{ page.text }}
+        </li>
+      {% else %}
+        <li class="ecl-pager__item">
+          <a class="ecl-pager__link" title="{{ page.title }}" href="{{ page.href }}">
+            <span class="ecl-u-sr-only" >{{ label_page }} </span>{{ page.text }}
+          </a>
+        </li>
+      {% endif %}
+    {% endfor %}
+
+    {# Print the right 'ellipsis'. #}
+    {% if max_page - current_page > 3 %}
       <li class="ecl-pager__item ecl-pager__item--ellipsis">…</li>
     {% endif %}
-    {% if _current_page != _max_page %}
+
+    {# Print the last page link. #}
+    {% if last %}
       <li class="ecl-pager__item ecl-pager__item--last">
-        <a class="ecl-pager__link" title="{{ _title_page }} {{ _max_page }}" href="{{ _fragment_url }}={{ _max_page }}">
-          <span class="ecl-u-sr-only" >{{ _label_page }} </span>{{ _max_page }}
+        <a class="ecl-pager__link" title="{{ last.title }}" href="{{ last.href }}">
+          <span class="ecl-u-sr-only" >{{ last.text }} </span>
+          {{ last.text }}
         </a>
       </li>
     {% endif %}
-    {% if _current_page != _max_page %}
+
+    {# Print the next page link. #}
+    {% if current_page != max_page %}
       <li class="ecl-pager__item ecl-pager__item--next">
-        <a class="ecl-pager__link" title="{{ _title_next }}" href="{{ _fragment_url }}={{ _current_page + 1 }}">{{ _label_next }}</a>
+        <a class="ecl-pager__link" title="{{ next.title }}" href="{{ next.href }}">{{ next.text }}</a>
       </li>
     {% endif %}
   </ul>
