patch-package
--- a/node_modules/@ecl/twig-component-footer-standardised/footer-standardised.html.twig
+++ b/node_modules/@ecl/twig-component-footer-standardised/footer-standardised.html.twig
@@ -131,6 +131,7 @@
                   {% endif %}
                   {% include '@ecl/link/link.html.twig' with _item|merge({
                     link: _item.link|default({})|merge({ type: 'standalone' }),
+                    icon: _item.icon|default({})|merge({ size: 'xs' }),
                     extra_classes: 'ecl-footer-standardised__link',
                   }) only %}
                   {% if _item.content_after is defined and _item.content_after is not empty %}
