patch-package
--- a/node_modules/@ecl/twig-component-footer-core/footer-core.html.twig
+++ b/node_modules/@ecl/twig-component-footer-core/footer-core.html.twig
@@ -5,6 +5,8 @@
   - "rows" (array of objects) (default: [])
     - "logo"
       - "alt" (string) alt attribute for the logo link
+      - "aria_label" (string) aria-label attribute for the logo link
+      - "title" (string) title attribute for the logo img
       - "path" (string) logo link path
       - "language" (string) Language code
       - "src_mobile" (string) Path the mobile logo (only for EU)
@@ -91,7 +93,7 @@
                 path: _sections.logo.path,
                 label: _label,
                 type: 'standalone',
-                aria_label: _sections.logo.title
+                aria_label: _sections.logo.aria_label,
               },
               extra_classes: 'ecl-footer-core__logo-link'
             } only %}
