patch-package
--- a/node_modules/@ecl/twig-component-card/card.html.twig
+++ b/node_modules/@ecl/twig-component-card/card.html.twig
@@ -112,7 +112,7 @@
       </ul>
     {% endif %}
     {% if _card.meta is not empty %}
-      <div class="ecl-card__meta">{{ _card.meta|join(" | ") }}</div>
+      <div class="ecl-card__meta">{{ _card.meta|safe_join(" | ") }}</div>
     {% endif %}
     {% if _card.title is not empty and _card.title.label is not empty %}
       <h1 class="ecl-card__title">
