patch-package
--- a/node_modules/@ecl/twig-component-file/file.html.twig
+++ b/node_modules/@ecl/twig-component-file/file.html.twig
@@ -113,7 +113,11 @@
     <div class="ecl-file__detail">
       <div class="ecl-file__detail-info">
       {% if _labels is defined %}
-        {{- _labelsMarkup -}}
+        {% for _label in _labels %}
+          <div class="ecl-file__label">
+            {% include '@ecl/label/label.html.twig' with _label only %}
+          </div>
+        {% endfor %}
       {% endif %}
       {% if _meta_detail_string is not empty %}
         <div class="ecl-file__detail-meta">
@@ -146,7 +150,11 @@
     <div class="ecl-file__info">
     {% if _variant == 'default' %}
       {% if _labels is defined %}
-        {{ _labelsMarkup }}
+        {% for _label in _labels %}
+          <div class="ecl-file__label">
+            {% include '@ecl/label/label.html.twig' with _label only %}
+          </div>
+        {% endfor %}
       {% endif %}
       {% if  _meta_detail_string is not empty %}
       <div class="ecl-file__detail-meta">
